econml/metalearners/_metalearners.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, *,
                  models,
                  categories='auto'):
         self.models = clone(models, safe=False)
+        self.categories = categories
```

Comment:
Note that if this attribute (which is now public) is set between calls to fit, the new value won't be used since the OneHotEncoder won't be updated.

Reply:
So maybe add a TODO to this file (if there isn't already one) that we need to move a bunch of metalearner logic to `fit` to be more like sklearn/ortholearner.